src/coreclr/jit/emitarm64.cpp

```diff
@@ -1259,6 +1276,8 @@ bool emitter::emitInsMayWriteToGCReg(instrDesc* id)
         case IF_DV_2B: // DV_2B   .Q.........iiiii ......nnnnnddddd      Rd Vn[]    (umov - to general)
         case IF_DV_2H: // DV_2H   X........X...... ......nnnnnddddd      Rd Vn      (fmov - to general)
 
+        case IF_SVE_DO_2A: // ........xx...... .....X.MMMMddddd -- SVE saturating inc/dec register by predicate count
```

Comment:
Why might these two groups write to GC Reg? (It's possible I've been missing some from my PRs)

Reply:
Ah sorry, didn't mean to push that in the first place. I don't think we should worry about touching `emitInsMayWriteToGCReg` for now. Once we start emitting these SVE instructions for actual codegen, we might need to come back to this method.